.vscode/launch.json

```diff
@@ -11,13 +11,9 @@
                 "-p",
                 "iOS",
                 "-n",
-                "HelloWorld",
+                "c/helloWorld",
                 "-t",
-                "sfdxdebug",
```

Comment:
A while ago I unknowingly modified this file, and now I'm reverting it back. Without this file reverted, if someone cloned our repo they would not be able to debug from within the VSCode Run tab b/c it would throw an error that `myConfig.json` is not found (and rightfully so).